front/src/modules/views/queries/create.ts

```diff
@@ -0,0 +1,13 @@
+import { gql } from '@apollo/client';
+
+export const CREATE_VIEW_FIELDS = gql`
```

Comment:
regarding creation/upsert. I wonder if we could assume that all workspaces have default view fields:
We could:
1)  Add them to seeds
2) Add them to the workspace creation flow (we are already provisioning some companies and people)
3)  run a script on production to add view fields to all existing workspaces

( for 2) and 3), maybe we should wait to have introduced view, viewSort and viewFilter models)

That's being said, it's great to have an upsert mechanism as it makes it more reliable